html-formatter/ruby/assets/index.mustache.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Cucumber</title>
```

Comment:
This file is duplicated 3 times over now. Perhaps we should copy it from the `html-formatter` directory.

Reply:
It should have been removed with commit 14a6e12, but it did not. Normally that's ok now.